go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/open-policy-agent/gatekeeper/v3
 
 go 1.21
 
+replace github.com/open-policy-agent/frameworks/constraint => /mount/d/go/src/github.com/open-policy-agent/frameworks/constraint
```

Comment:
commenting so we don't forget updating this

Reply:
Removed local vendoring of frameworks.